MPF/ViewModels/MainViewModel.cs

```diff
@@ -1532,7 +1549,10 @@ private void SystemTypeComboBoxSelectionChanged(object sender, SelectionChangedE
         private void UpdateVolumeLabelClick(object sender, RoutedEventArgs e)
         {
             if (_canExecuteSelectionChanged)
-                InitializeUIValues(removeEventHandlers: true, rescanDrives: false);
+                if (App.Options.FastUpdateLabel)
+                    FastUpdateLabel();
+                else
+                    InitializeUIValues(removeEventHandlers: true, rescanDrives: false);
```

Comment:
Nitpicky thing - Can you wrap these 4 lines in curly braces? even though it's mostly apparent, I prefer making it obvious that these are all under the first `if`.